public/app/features/explore/state/epics/runQueriesBatchEpic.ts

```diff
@@ -115,26 +112,6 @@ export const runQueriesBatchEpic: Epic<ActionOf<any>, ActionOf<any>, StoreState>
             }
 
             if (state === LoadingState.Streaming) {
-              if (event.request && event.request.range) {
```

Comment:
This was used to shift the graph during streaming, so we need to shift the TimeRange somehow for that part of Explore somehow. Otherwise you'll have streaming in graphs but the graphs X-Axis will not change. I think TestData is still broken as of now but you could try the TestData and Streaming Client.

Reply:
And where I can see that in action? I mean I do not see a way to show the graph when live streaming now.

Reply:
Unfortunately the Streaming Client is broken in Explore as of now :(